lib/WorseReflection/Bridge/TolerantParser/Reflection/ReflectionTrait.php

```diff
@@ -59,6 +60,11 @@ public function members(): ReflectionMemberCollection
         return $this->members;
     }
 
+    public function constants(): ReflectionConstantCollection
+    {
+        return $this->members()->constants();
```

Comment:
is this a new feature?

Reply:
Yeah, I probably should create a new PR for that. But I think it would be easier if the `ReflectionClassLike` interface would have the `constants()` method since all implementation of `ClassLike` except have it, except the enum. And there we could just return an empty collection.